### PR TITLE
fix: Workspace schedule button on responsive

### PR DESCRIPTION
**Pull request by BrunoQuaresma**
After:
Screen Shot 2022-07-28 at 11 21 44

With + and - buttons:
Screen Shot 2022-07-28 at 11 53 43

Before:
Screen Shot 2022-07-28 at 11 02 34

### CODE REVIEW

File: site/src/components/WorkspaceScheduleButton/WorkspaceScheduleButton.stories.tsx [hunk 1]
```diff
@@ -87,3 +87,9 @@ CannotEdit.args = {
   },
   canUpdateWorkspace: false,
 }
+
+export const SmallViewport = Template.bind({})
+SmallViewport.args = WorkspaceOffShort.args
+SmallViewport.parameters = {
+  chromatic: { viewports: [320] },
```
Comment (presleyp) [inline site/src/components/WorkspaceScheduleButton/WorkspaceScheduleButton.stories.tsx:94]: This looks useful, how does it work?

**Review by presleyp — APPROVED** [review]

File: site/src/components/WorkspaceScheduleButton/WorkspaceScheduleButton.stories.tsx (hunk 1 above)
Comment (BrunoQuaresma) [reply site/src/components/WorkspaceScheduleButton/WorkspaceScheduleButton.stories.tsx:94]: We basically are saying to chromatic to test this snapshot using the 320px viewport. Unfortunately, looks like a Chromatic thing not a Storybook native option.

**Review by Kira-Pilot — APPROVED** [review]

File: site/src/components/WorkspaceScheduleButton/WorkspaceScheduleButton.stories.tsx (hunk 1 above)
Comment (Kira-Pilot) [reply site/src/components/WorkspaceScheduleButton/WorkspaceScheduleButton.stories.tsx:94]: Looks like maybe Chromatic isn't registering this one?

**Review by AbhineetJain — APPROVED** [review]
Looks good! Great job 👏

**Comment by BrunoQuaresma** [comment]
Fixes #3267

**Pull request merged**